docs/content/standards/display.mdx

```diff
@@ -107,14 +107,15 @@ module examples::my_hero {
         // Commit first version of `Display` to apply changes.
         display::update_version(&mut display);
 
-        transfer(publisher, sender(ctx));
-        transfer(display, sender(ctx));
+        transfer::public_transfer(publisher, sender(ctx));
+        transfer::public_transfer(display, sender(ctx));
     }
 
     /// Anyone can mint their `Hero`!
-    public fun mint(name: String, img_url: String, ctx: &mut TxContext): Hero {
+    public fun mint(name: String, img_url: String, ctx: &mut TxContext) {
         let id = object::new(ctx);
-        Hero { id, name, img_url }
+        let hero = Hero { id, name, img_url };
+        transfer::public_transfer(hero, sender(ctx));
```

Comment:
I wouldn't change this signature. This is how we do things today - return so the sender can decide whether to transfer or not.